docs/technical-details/release-notes/4.tucker/README.md

```diff
@@ -2,13 +2,15 @@
 
 HarperDB version 4 ([Tucker release](./tucker.md)) represents major step forward in database technology. This release line has ground-breaking architectural advancements including:
 
-## [4.0](./4.0.0.md)
-* New clustering technology that delivers robust, resilient and high-performance replication
-* Major storage improvements with highly-efficient adaptive-structure modified MessagePack format, with on-demand deserialization capabilities
+## [4.4](./4.4.0.md)
+* Native replication (codename "Plexus") which is faster, more efficient, secure, and reliable than the previous replication system and provides provisional sharding capabilities with a foundation for the future
+* Computed properties that allow applications to define properties that are computed from other properties, allowing for composite properties that are calculated from other data stored in records without requiring actual storage of the computed value
```

Comment:
@jcohen-hdb  @kriszyp 
 
 Define properties that are automatically calculated based on other fields in your records. This allows for dynamic, composite values without needing to store the computed result directly in the database.